pdf/renderer.go

```diff
@@ -901,15 +892,10 @@ func (w *pdfPageWriter) WriteText(TJ ...interface{}) {
 			var rPrev rune
 			for j, r := range val {
 				if i < j {
-					i0, err0 := w.font.sfnt.GlyphIndex(&sfntBuffer, rPrev)
-					i1, err1 := w.font.sfnt.GlyphIndex(&sfntBuffer, r)
-					if err0 == nil && err1 == nil {
-						kern, err := w.font.sfnt.Kern(&sfntBuffer, i0, i1, toI26_6(units), font.HintingNone)
-						if err == nil && kern != 0.0 {
-							write(val[i:j])
-							fmt.Fprintf(w, " %d", -int(fromI26_6(kern)*1000.0/units+0.5))
```

Comment:
In general `units` does not have to be `1000` (I've seen `2000` I think as well), so I think this was an oversimplification.

Reply:
All right, I changed it back